examples/demo-app/README.md

```diff
@@ -1,23 +1,29 @@
 # Demo App
 
-This is the src code of kepler.gl demo app.
+This is the src code of kepler.gl demo app. You can copy this folder out and run it locally.
```

Comment:
I don't think users can simply copy this directory and run the example outside of kepler.gl folder.

There are node dependencies fetched from root node_modules folder based on webpack project settings

Reply:
we have 2 webpack file, the one included in the demo-app folder should be sufficient to run this. and if we are missing anything we should add it to its package.json